adminPanel/editform.php

```diff
@@ -2,7 +2,9 @@
 $db = new PDO("mysql:host=192.168.20.20;dbname=PortfolioProjects", 'root', '');
 $db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 $sql = "SELECT `title`, `link`, `image` FROM `projects` WHERE `id` = :id;";
-$id = $_GET['id'];
+if (!empty($_GET['id'])) {
```

Comment:
you need to put the prepare and execute inside this if as well, otherwise it will error.

probably add an else that redirects to the admin panel